scripts/novel_mutations.py

```diff
@@ -9,7 +9,9 @@
 import argparse
 import sys
 import math
+import warnings
 
+warnings.simplefilter(action='ignore', category=FutureWarning)
```

Comment:
We may want to document which warning we are ignoring somewhere. Could be a comment in the code or maybe just attached to this issue.

Reply:
The warning is for concatenating with empty dataframes :)